### ISSUE TITLE
SyntaxError on python 3.7

### ISSUE THREAD

**Issue by vfdev-5** [polyaxon/polyaxon-cli#7]
Hi, just installed locally `polyaxon-cli` with `pip install` following the [documentation](https://docs.polyaxon.com/installation/install_polyaxon_cli/#install) and when I run `polyaxon --help` I get the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/polyaxon", line 7, in <module>
    from polyaxon_cli.main import cli
  File "/usr/local/lib/python3.7/site-packages/polyaxon_cli/main.py", line 13, in <module>
    from polyaxon_cli.cli.experiment import experiment
  File "/usr/local/lib/python3.7/site-packages/polyaxon_cli/cli/experiment.py", line 338
    ctx.invoke(upload, async=False)
                           ^
SyntaxError: invalid syntax
```

**Comment by mouradmourafiq** [comment]
@vfdev-5 I just fixed that, the fix will be part of the coming release (today or tomorrow).

The other commands will work, you can ignore this bug until next release, sorry for that.

**Comment by vfdev-5** [comment]
@mouradmourafiq thank you !

**Comment by mouradmourafiq** [comment]
The fix is part of 0.2.9.
